dace/transformation/interface/vscode/__main__.py

```diff
@@ -1,6 +1,32 @@
 import json
 from argparse import ArgumentParser
 
+def get_exception_message(exception):
+    if hasattr(exception, 'message'):
+        return exception.message
+    else:
+        return str(exception)
+
+def load_sdfg_from_json(json):
+    # We lazy import SDFGs, not to break cyclic imports, but to avoid any large
+    # delays when booting in daemon mode.
+    from dace.sdfg import SDFG
+    try:
+        sdfg = SDFG.from_json(json)
+        error = None
+    except Exception as e:
+        error = {
+            'error': {
+                'message': 'Failed to parse the provided SDFG',
+                'details': get_exception_message(e),
```

Comment:
I would just use `str(e)` and include the type there too, i.e., `'%s: %s' % (type(e).__name__, e)`

Reply:
Very good idea, the received message is a lot cleaner this way - the `get_exception_message` method has been changed accordingly.